packages/editor/src/components/post-slug/index.js

```diff
@@ -24,8 +24,7 @@ function PostSlugControl() {
 
 	return (
 		<TextControl
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
This doesn't appear anywhere in the app anymore, but code-wise it looks safe to change.